src/App.css

```diff
@@ -57,3 +58,11 @@
     grid-template-columns: repeat(4, 1fr);
   }
 }
+
+.skillsButton {
```

Comment:
camelCase has no place in class names, you're following BEM, so the general rule is this: `cool-block__cool-element--cool-modifier`.